apps/src/code-studio/pd/workshop_dashboard/components/workshop_enrollment_school_info.jsx

```diff
@@ -47,6 +57,31 @@ export default class WorkshopEnrollmentSchoolInfo extends React.Component {
     this.props.onDelete(pendingDeleteId);
   }
 
+  handleScholarshipStatusChange(enrollment, selection) {
+    let pendingScholarshipUpdates = this.state.pendingScholarshipUpdates;
+    pendingScholarshipUpdates.push(enrollment.id);
```

Comment:
Not really a problem here since you're doing an immediate state update, but in theory you shouldn't use `push` here because it modifies `this.state.pendingScholarshipUpdates` in-place, which you're not supposed to do to React state.  I'd write something like:
```jsx
this.setState({
  pendingScholarshipUpdates: [
    ...this.state.pendingScholarshipUpdates,
    enrollment.id
  ]
});
```